atm.js

```diff
@@ -1,21 +1,101 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+function overDraft(){
+  if (savingsBalance > 0) {
+    var depositAmount = parseFloat($(checkingInput).val());
+    savingsBalance -= depositAmount;
+    checkingInput.val("");
+    savingsDisplay.html("$"+ savingsBalance);
+    savingsDisplay.css("background-color","red");
```

Comment:
Think about the difference between assigning an attribute and setting a class here.